### ISSUE TITLE
Polar2Grid V3.0 HDF5 writer creates reflectances using different scaling than Version 2.3

### ISSUE THREAD

**Issue by kathys** [ssec/polar2grid#418]
When creating an HDF5 file from different  VIIRS bands, the reflectances in the Version 3.0 software look like they are created on a 0-100 scale as opposed to previous version 0-1. scale.  Since the values are stored as floats, it makes a lot more sense for these to be on the 0-1. scale, especially for end users who are using the P2G Version 2.3 HDF5 writer now.

**Comment by djhoese** [comment]
Oh no. This is going to be a problem I think. Satpy produces reflectances as "%". I'm not sure of an easy way to get around this.

> Since the values are stored as floats, it makes a lot more sense for these to be on the 0-1. scale

What do you mean by this?

**Comment by kathys** [comment]
Yeah, that is a stupid comment.  Please ignore.

**Comment by djhoese** [comment]
@kathys I don't think we've talked about this in our meetings recently and probably should. Supporting unit conversion is not a simple change to this writer, but isn't impossible either.

**Comment by kathys** [comment]
Okay, let's discuss it at the next meeting.  If someone has an automated process set up to convert the files, this would be a big change on the user side.

**Comment by djhoese** [comment]
I need to verify that the units produced by the AWIPS tiled writer don't also suffer from this problem (make sure they say `%` instead of `1`, AWIPS should handle whichever it is). ...verify on AWIPS also.

**Comment by djhoese** [comment]
Ok, I will add a flag to turn off a new default behavior: reflectances with units `%` and standard_name `toa_bidirectional_reflectance` will be divided by 100 before being written to disk. This needs to be done for the HDF5 and the Binary writers.

**Comment by djhoese** [comment]
As our meeting today continued discussion on this issue we realized that a flag/option/conversion like this would also be needed for geotiff writing when saving to floating point which complicates the matter a bit. The HDF5 and binary writers are "our" writers and don't live in Satpy at all, the geotiff is part of Satpy. It isn't impossible to add this conversion to that writer, but it gets a little uglier than the other two.

We've decided to think about it for a while: add an automatic conversion from % to 0-1 range or just document the change and see who that effects.
